saleor/graphql/shop/types.py

```diff
@@ -76,6 +77,8 @@ class Shop(graphene.ObjectType):
         required=True)
     default_currency = graphene.String(
         description='Default shop\'s currency.', required=True)
+    enabled_gateways = graphene.List(
```

Comment:
It might not be obvious that we are talking about the payments gateways. I’d opt for payment_gateways or available_payment_gateways 

(Don’t feel like “enabled” fits well in this context, as the API is user facing we should talk about the gateways that are available to use, not enabled by the store’s admin)